### PR TITLE
Feature/fix volume color

### PR DESCRIPTION
**Pull request by atecon**

### CODE REVIEW

File: tests/run_tests.inp [hunk 1]
```diff
@@ -0,0 +1,81 @@
+set verbose off
+clear
+
+set assert stop
+include assertion.gfn
+include "./src/candlesticks.inp" --force
```
Comment (deferoci) [inline tests/run_tests.inp:6]: Misstyped path
`include "../src/candlesticks.inp" --force`

File: src/candlesticks.inp [hunk 2]
```diff
@@ -6,13 +6,54 @@ function void run_checks (const list L,
...
+function void check_input_consistency (const list L)
+    /* Check for the following ordering: Low <= (Open, Close) <= High. */
+
+    series open_ = L[1]
+    series close_ = L[2]
+    series low_ = L[3]
+    series high_ = L[4]
+
+    low_exceeds_high(low_, high_)
+    low_exceeds_close(low_, close_)
```
Comment (deferoci) [inline src/candlesticks.inp:27]: Add
`low_exceeds_open(low_, open_)`

File: src/candlesticks.inp [hunk 3]
```diff
@@ -6,13 +6,54 @@ function void run_checks (const list L,
...
+            "Inconsistency: Series 'Low' exceeds 'High'.")
+end function
+
+function void low_exceeds_close (const series low_,
+                                const series close_)
+    errorif(sum(low_ > close_),
+            "Inconsistency: Series 'Low' exceeds 'Close'.")
+end function
+
+
```
Comment (deferoci) [inline src/candlesticks.inp:63]: Add
function void low_exceeds_open (const series low_,
                                const series open_)
    errorif(sum(low_ > open_),
            "Inconsistency: Series 'Low' exceeds 'Open'.")
end function

**Review by deferoci — CHANGES_REQUESTED** [review]

File: tests/run_tests.inp (hunk 1 above)
Comment (atecon) [reply tests/run_tests.inp:6]: Not really, the test is executed within the main folder `./candlesticks` by means of the command `./run_tests.sh` on a LINUX machine.
You can execute the script locally but then you would have to set the working directory. For instance, by adding to the file `./tests/run_tests.inp` file.
~~~
set workdir "PATH/candelsticks"
include "../src/candlesticks.inp" --force
~~~

However, the reason to go via the shell script is to run automatic tests in a later stage (continuous integration).

File: src/candlesticks.inp (hunk 2 above)
Comment (atecon) [reply src/candlesticks.inp:27]: Thanks, I added this case.

File: src/candlesticks.inp (hunk 3 above)
Comment (atecon) [reply src/candlesticks.inp:63]: Done

**Review by deferoci — APPROVED** [review]
Everything ok

**Pull request merged**